kaldb/src/main/java/com/slack/kaldb/server/ArmeriaService.java

```diff
@@ -57,10 +56,16 @@ private ArmeriaService(Server server, String serviceName) {
   public static class Builder {
     private final String serviceName;
     private final ServerBuilder serverBuilder;
+    private final Duration requestTimeout;
     private final List<SpanHandler> spanHandlers = new ArrayList<>();
 
-    public Builder(int port, String serviceName, PrometheusMeterRegistry prometheusMeterRegistry) {
+    public Builder(
+        int port,
+        Duration requestTimeout,
```

Comment:
This probably shouldn't be an argument on the constructor, but instead a builder method.

The only arguments that are expected to go to a builder's constructor are those that are required (when it's not a step-builder). Request timeout isn't a required argument, and if it's not provided we can leave it unset and fallback to the default Armeria behavior.